Sources/Logging/Logging.swift

```diff
@@ -229,7 +465,11 @@ extension Logger {
     }
 }
 
-/// Ships with the logging module, used to multiplex to multiple logging handlers
+/// A pseudo-`LogHandler` that can be used to send messages to multiple other `LogHandler`s.
```

Comment:
Maybe avoid calling it "pseudo"?  

Maybe just proxying or "A `LogHandler` forwarding all messages to multiple wrapped handlers." (also avoiding the repetition of LogHandler word)